packages/web3-utils/src/ethjsUnit.js

```diff
@@ -0,0 +1,190 @@
+// This file was internalized from silentcicero/ethjs-unit
+// in order to reduce the bundle size. Package shared a
+// bn.js dependency which wasn't being updated resulting
+// in duplicate versions of that dep included in webpacks
+
+/*
+The MIT License (MIT)
+
+Copyright (c) 2016 Nick Dodson
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.*/
+
+/*
+Primary Attribution
+Richard Moore <ricmoo@me.com>
+https://github.com/ethers-io
+Note, Richard is a god of ether gods. Follow and respect him, and use Ethers.io!
+*/
+
+const BN = require('bn.js');
```

Comment:
Isn't ethers a dependency? Why is this duplicated?

Reply:
That's a good question...discussed a bit below. This code is a few years old now and swapping it for ethers will take a little additional research.